Cargo.toml

```diff
@@ -174,6 +176,8 @@ trustify-module-user = { path = "modules/user" }
 trustify-server = { path = "server", default-features = false }
 trustify-test-context = { path = "test-context" }
 trustify-ui = { git = "https://github.com/trustification/trustify-ui.git", branch = "publish/main" }
+query = {path = "query" }
```

Comment:
Might want to consider renaming this to `trustify-query` to align with the others.